Keas.Mvc/ClientApp/components/Keys/CreateKey.tsx

```diff
@@ -1,176 +1,197 @@
-import * as PropTypes from "prop-types";
```

Comment:
what are you changing here? line endings? formatting? try to make sure it's clear what's changed when you push files--the way it is now, it's hard to tell if you've changed anything